lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/BuildProvenUserIdentityDetailsHandler.java

```diff
@@ -165,22 +164,10 @@ private NameAndDateOfBirth getProvenIdentityNameAndDateOfBirth(
                         500, ErrorResponse.FAILED_TO_GENERATE_IDENTIY_CLAIM);
             }
 
-            Name name = mapper.convertValue(identityClaim.get().getName().get(0), Name.class);
             BirthDate birthDate =
                     mapper.convertValue(identityClaim.get().getBirthDate().get(0), BirthDate.class);
```

Comment:
Would this suit being a BirthDate class method, if getFullName is going to be?

Reply:
I'm not sure that this call to `mapper` is actually doing anything at all here, so the only thing I think you could refactor would be to put a method on the IdentityClaim to do the `get(0)`.
I believe that this is going to be properly refactored in a later ticket so I'm planning to leave it alone.